Testreglar/3.1.1/App/app-3.1.1a.json

```diff
@@ -0,0 +1,106 @@
+	"id": "app-3.1.1a",
+	"testlabId": 480,
+	"versjon": "1.0",
+	"type": "App",
+	"spraak": "nb",
+	"kravTilSamsvar": "<ul>\r\n<li>Appsiden har en gyldig språkkode og</li>\r\n<li>Språkkoden som er angitt stemmer med hovedspråket på den aktuelle appsiden</li>\r\n</ul>",
+	"side": "2.1",
+	"element": "Side",
+	"kolonner": [
+		{
+			"title": "2.2"
+		},
+		{
+			"title": "2.3"
+		},
+		{
+			"title": "3.2"
+		}
+	],
+	"steg": [
+		{
+			"stegnr": "2.1",
+			"spm": "Hvilken appside tester du?",
+			"ht": "<p>Beskriv appsiden med få stikkord, eller velg i listen.</p>",
+			"type": "tekst",
+			"label": "Appside:",
+			"datalist": "Sideutvalg",
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.2"
+				}
+			}
+		},
+		{
+			"stegnr": "2.2",
+			"spm": "Er det mulig å sveipe til innhold på appsiden?",
+			"ht": "<p>Slik tester du:</p>\n<ul>\n<li>Aktiver skjermleser\n<ul>\n<li>iOS: VoiceOver</li>\n<li>Android: Talkback</li>\n</ul>\n</li>\n<li>Åpne appsiden du tester</li>\n<li>Sjekk om det er mulig å sveipe til innhold på appsiden</li>\n</ul>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "2.3"
+				},
+				"nei": {
+					"type": "avslutt",
+					"fasit": "Ikkje testbart",
+					"utfall": "Det er ikke mulig å sveipe til innhold på appsiden."
+				}
+			}
+		},
+		{
+			"stegnr": "2.3",
+			"spm": "Hva er hovedspråket på appsiden?",
+			"ht": "<p>Velg det språket som alt eller størsteparten av innholdet på appsiden er har.</p>\n<ul>\n<li>Dersom det er vanskelig å vurdere hvilket språk som er mest brukt, er hovedspråket det språket som er brukt først.</li>\n<li>Dersom språket ikke finnes i listen, velger du alternativet Annet og går videre.</li>\n</ul>",
+			"type": "radio",
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "3.1"
+				}
+			},
+			"svarArray": [
+				"Bokmål",
+				"Nynorsk",
+				"Nordsamisk",
+				"Sørsamisk",
+				"Lulesamisk",
+				"Engelsk",
+				"Annet"
+			]
+		},
+		{
+			"stegnr": "3.1",
+			"spm": "Velg et annet språk i operativsystemet, enn hovedspråket på appsiden",
+			"ht": "<ul>\n<li>Sett språket i operativsystemet til engelsk hvis hovedspråket i appen er norsk eller omvendt</li>\n<li>Slik endrer du språk i operativsystemet\n<ul>\n<li><strong>iOS:</strong> Innstillinger -&gt; Generelt -&gt; Språk og område -&gt; Sett språket i operativsystemet til engelsk eller omvendt</li>\n<li><strong>Android</strong>: Innstillinger -&gt; Generell styring -&gt; Språk -&gt; Legg til språk -&gt; Sett språket i operativsystemet til engelsk eller omvendt\n<ul>\n<li>Gå til Tekst til tale -&gt; Språk -&gt; Velg Bruk systemspråk.</li>\n</ul>\n</li>\n</ul>\n</li>\n</ul>",
+			"type": "instruksjon",
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "3.2"
+				}
+			}
+		},
+		{
+			"stegnr": "3.2",
+			"spm": "Leser skjermleser opp innhold på hovedspråket, med riktig uttale?",
```

Comment:
Riktig uttale kan vel ikke legges til som en feil? Isåfall bør det spesifiseres at enkeltord kan leses opp med feil uttale. Talkback spesielt har flere ord den leser opp med feil uttale selvom det leser opp alt annet helt riktig.

Reply:
Enig i at dette bør være tydeligere.

Reply:
Det må være uttale som åpnebart stammer fra feil språkvalg.